pkg/transformer/openshift/openshift.go

```diff
@@ -171,6 +171,10 @@ func (o *OpenShift) initImageStream(name string, service kobject.ServiceConfig)
 
 // initBuildConfig initialize Openshifts BuildConfig Object
 func initBuildConfig(name string, service kobject.ServiceConfig, composeFileDir string, repo string, branch string) *buildapi.BuildConfig {
+	envList := []kapi.EnvVar{}
+	for i, v := range service.BuildArgs {
+		envList = append(envList, kapi.EnvVar{Name: i, Value: v})
+	}
```

Comment:
nit: how about using something like `envName` or `envVal` instead of `i` and `v`?